x-pack/filebeat/docs/inputs/input-aws-s3.asciidoc

```diff
@@ -261,7 +261,13 @@ The default is `10 MiB`.
 ==== `max_number_of_messages`
 
 The maximum number of SQS messages that can be inflight at any time. Defaults
-to 5.
+to 5. When processing large amount of large size S3 objects and each object has
+large amount of events, if this parameter sets too high, it can cause the input
```

Comment:
> When processing large amount of large size S3 objects and each object has large amount of events, if this parameter sets too high, it can cause the input to process too many messages concurrently, overload the agent and cause ingest failure.

Perhaps something like:
Setting this parameter too high can overload Elastic Agent and cause ingest failures in situations where the SQS messages contain many S3 objects or the S3 objects themselves contain large numbers of messages.

Or minimally:
> When processing a large number of large size S3 objects that each have a large number of events, if this parameter is set too high, it can cause the input to process too many messages concurrently, overloading the agent and causing ingest failure.

Reply:
@strawgate Thanks for the review!! Ugh sorry this is a backup PR. Let me change it in a main branch in a separate PR and backport it into these branches.